capstone/src/main/java/com/google/sps/servlets/GroupMemberServlet.java

```diff
@@ -0,0 +1,92 @@
+package com.google.sps.servlets;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import com.google.gson.Gson;
+import com.google.sps.Objects.response.MemberResponse;
+import error.ErrorHandler;
+
+@WebServlet("/group-member")
+
+public class GroupMemberServlet extends HttpServlet {
+
+  @Override
+  // Adds a new member to a group 
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
+    String userId = this.getUserId(response);
```

Comment:
To clarify, this adds the current user to a specified group.. so eventually would this be called when the user enters a code to join a group?

Reply:
yeah this adds a user to a group for later on